test/e2e/main_test.go

```diff
@@ -148,10 +165,10 @@ func SetupApps(t resource.Context, i istio.Instance, apps *EchoDeployments) erro
 				},
 				{
 					Replicas: 1,
-					Version:  "v2",
+					Version:  "v1",
```

Comment:
What was written before is correct. 

A subset represents a deployment corresponding to a service. A service can have multiple subsets representing deployments of different versions.